packages/gestalt/src/Checkbox.js

```diff
@@ -9,6 +9,7 @@ import Icon from './Icon.js';
 type Props = {|
   checked?: boolean,
   disabled?: boolean,
+  error?: boolean,
```

Comment:
does it make more sense to call this `hasError` like `TextField` and `TextArea`? whats the long term plan for these props? i think one consistent name is clearer to users